common/remote/sockfile.cpp

```diff
@@ -1,4 +1,4 @@
-/*##############################################################################
+﻿/*##############################################################################
```

Comment:
Any idea what changed here??

Reply:
The new version adds an EF BB BF header (means UTF8 encoding). It was probably added by some editor. Will remove it since no other cpp/hpp file has it.